lib/components/FairnessCard.jsx

```diff
@@ -56,6 +59,23 @@ export const FairnessCard = (props) => {
           />
         </div>
       </InputLabel>
+      <Collapse title='Advanced Settings'>
+        <TextInputGroup
+          id='_numberOfWiners'
+          containerClassName='w-full sm:w-1/2 sm:ml-2'
+          label='Number of winners'
+          required
+          type={TextInputGroupType.number}
+          max={100}
```

Comment:
We better use 10 here instead of 100. The gas costs could be above the 8 or 10 million block gas limit if we have to loop through 100 times!